documentation/tutorial-newapp.asciidoc

```diff
@@ -138,10 +138,7 @@ PrincipalAccessControlProviderImpl.java|Implementation of PrincipalAccessControl
 |=============================================
 |*File* | *Description*
 |api.UseCase.java|Annotation to mark all use-cases.
-|api.to.BinaryObjectEto.java|ETO for a BinaryObject.
 |base.AbstractUc.java|Abstract base class for any use case in the application.
-|base.UcManageBinaryObject.java|Use case for managing BinaryObject.
-|impl.UcManageBinaryObjectImpl.java|Implementation of the UcManageBinaryObject interface.
 |impl.UsermanagementDummyImpl.java|Implementation of Usermanagement.
```

Comment:
Do we still ship `UsermanagementDummyImpl`? I assume this can also be removed. Can also be done afterwards as not directly related to this PR...

Reply:
`UsermanagementDummyImpl`has nothing to do with the binary thing.
There are even more things we could cleanup in the template (e.g. batches).